CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## \[Unreleased\]
 
+### Added
+
+- doc on the model loading page ()
```

Comment:
Why the "()"? It should be the PR number?

Reply:
Thanks, I wrote the changelog before having the number of the pr and forgot to update :)